frontend/src/i18n/en-US.json

```diff
@@ -108,5 +108,12 @@
   "vaultList.title": "Vaults",
   "vaultList.empty.title": "No vaults",
   "vaultList.empty.description": "Get started by creating a new vault.",
-  "vaultList.createVault": "Create Vault"
+  "vaultList.createVault": "Create Vault",
+  "vaultList.filter": "Filter",
+  "vaultList.filter.entry.allVaults": "All vaults",
+  "vaultList.filter.entry.ownedByMe": "Owned by me",
```

Comment:
Shouldn't this rather be something like this?

```suggestion
  "vaultList.filter.entry.accessibleByMe": "Vaults I have access to",
```

Reply:
Hmm you're right but `Vaults I have access to` is quiet long
![Screenshot from 2022-09-28 15-49-03](https://user-images.githubusercontent.com/1786772/192796651-0c390151-b710-443d-a5bc-56b73b24b2ac.png)

What do you think about `Accessible vaults`?
![Screenshot from 2022-09-28 15-53-51](https://user-images.githubusercontent.com/1786772/192797388-318d5639-2e06-43f9-b27d-a7cf6ef8e60a.png)